app/src/main/res/layout/fragment_main.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="eu.depau.etchdroid.ui.fragments.main.MainViewModel" />
+    </data>
+
```

Comment:
I don't think it can be necessary, but maybe can be used for `Beta` label of the button or any other feature.